test/integration/contracts/mixins/with-events.spec.ts

```diff
@@ -92,8 +101,8 @@ beforeAll(async () => {
 	});
 });
 
-afterAll(() => {
-	return testUtils.destroyContext(ctx);
+afterAll(async () => {
+	await testUtils.destroyContext(ctx);
```

Comment:
We don't need `async`/`await` here, we can just return `testUtils.destroyContext(ctx);` and have the same result.

Reply:
I updated all the `afterAll()` definitions to make them consistent; in the current version some use `return` , and others the `async/await` pattern as in this example.
I can update them to use any of the patters, so if the `return` seems more natural I can use that one

Reply:
I'll update this on my end today. It's not a big deal either way, but I'd like to keep things uniform across repos if possible.